workspace/src/common/launch/art_localization_launch/launch/art_localization.launch.py

```diff
@@ -48,7 +48,7 @@ def generate_launch_description():
     AddLaunchArgument(ld, "art_localization/input/magnetometer", "/sensing/magnetometer/data")
     AddLaunchArgument(ld, "art_localization/input/gyroscope", "/sensing/gyroscope/data")
     AddLaunchArgument(ld, "art_localization/input/accelerometer", "/sensing/accelerometer/data")
-    AddLaunchArgument(ld, "art_localization/output/vehicle_state", "/vehicle/state")
+    AddLaunchArgument(ld, "art_localization/output/vehicle_state", "/vehicle/filtered_state")
```

Comment:
Doesn't the control node only subscribe to /vehicle/state? Or what's the point of the localization node if no one is subscribing to `filtered_state`? Assuming that's the case since you didn't change any subscriber topics in this PR.

Reply:
Right now both Chrono and the SE node publish to /vehicle/state. I didn't change the PP and Control nodes because they don't actually do anything with the info atm. In feature/wpts_mpc branch, they do stuff with the information but are currently using the ground truth, so want to be listening to chrono. Makes sense to name SE output filtered, as it is not the true state. We could also rename the Chrono output to /vehicle/ground_truth_state...

Reply:
We should then remove these subscribers all together then and keep /vehicle/state. `art_localization` is a replacement for ground truth anyways, correct? I would think ground truth should publish to `/sensing/gps/data` or something, not `/vehicle/state`.

Reply:
> We should then remove these subscribers all together then and keep /vehicle/state. `art_localization` is a replacement for ground truth anyways, correct? I would think ground truth should publish to `/sensing/gps/data` or something, not `/vehicle/state`.

Yes, this is something I was hoping to merge in with the #2  merges. Creating a parameter that allows the user to choose which state estimation algorithm is used, or if they just want to use ground truth, and then 'art_control' and 'art_path_planning' always listen to 'art_localization'. But this doesn't seem related to the problem that Chrono and 'art_localization' are publishing to the same topic at the same time.

Reply:
ohhh okay, I didn't know it was because of Chrono. Sounds good.

Reply:
Rework first ya, then address the autonomy stack structure after.